static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain.js

```diff
@@ -101,11 +101,15 @@ define([
                 || _.contains(['politics', 'world', 'business', 'commentisfree'], config.page.section.toLowerCase()) ? 'sections' : 'all';
         },
 
+        identityPolicy: function () {
```

Comment:
Yes, I am going to do that. I wanted to do it for previous Outbrain PR already but it felt that I am doing to many things in one PR. I like to keep it separated. New plan is to do it after this PR as this should be the last 'test' one touching policies.